timeline-chart/src/layer/time-graph-chart-arrows.ts

```diff
@@ -64,4 +66,10 @@ export class TimeGraphChartArrows extends TimeGraphChartLayer {
         }
     }
 
+    destroy() : void {
+        super.destroy();
```

Comment:
In addArrows() above, when we replace all the arrows with a new list, each old arrow is destroyed. But they are not removed as children, so when super.destroy() is called here, they will be destroyed a second time, which throws an exception.

The old arrows should be removed with removeChildren(), this will remove and destroy all of them in one call.

Reply:
sounds good. I'll fix it